<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcome</class>
 <widget class="QMainWindow" name="welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1824</width>
    <height>1152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>welcome</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="userpicture">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>425</y>
      <width>170</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>480</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(4, 136, 172);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Sitka Small&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome To Universe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>440</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>500</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_out">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>1000</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>550</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1824</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>commandLinkButton</tabstop>
  <tabstop>pushButton_out</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
